static/js/containers/LoginButton.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import DropdownButton from 'react-bootstrap/lib/DropdownButton';
+import { LinkContainer } from 'react-router-bootstrap';
 import MenuItem from 'react-bootstrap/lib/MenuItem';
 import Link from 'react-router/lib/Link';
```

Comment:
oof nvm it's waaaay loud, it complains about this:

```
`import React from 'react';`

class Foo extends React.Component {}
```

says react isn't used...